src/_64FF00/PurePerms/PurePerms.php

```diff
@@ -457,7 +457,7 @@ public function getOnlinePlayersInGroup(PPGroup $group)
      */
     public function getPermissions(IPlayer $player, $levelName)
     {
-        // TODO: Fix this damn shit
```

Comment:
Tbh i can swear whenever the ~~fuck~~ i like in my code. It may be needless but isint playing Minecraft needless? You could be outside in the real world with real people other than staying inside playing games.